### ISSUE TITLE
Searching in specified path only works when another folder is already open

### ISSUE THREAD

**Issue by Sorlaq** [microsoft/vscode#72938]
- VSCode Version: 1.33.1
- OS Version: Windows_NT x64 10.0.17134

Steps to Reproduce:

1. Close folder (Ctrl+K F)
2. Do a search on specified path through Ctrl+Shift+F

![searchbad](https://user-images.githubusercontent.com/23391094/56837696-97e2cc80-687b-11e9-9d4f-078dbd9dc280.png)

With literally any folder on my system opened search works:

![searchgood](https://user-images.githubusercontent.com/23391094/56837711-a630e880-687b-11e9-9933-e7d316ec96d1.png)

I'm not sure if this is intended behavior, as the message suggests that it shouldn't work if I don't have _that particular_ folder currently opened anyways, and yet it does if I open anything.

Does this issue occur when all extensions are disabled?: Yes

**Comment by tillig** [comment]
I was just about to file an issue to find out if searching an arbitrary folder / folder tree was even possible since the note says only the currently opened folder will be searched. This is one of the things Sublime has that I was hoping VS Code had. I never tried opening _any ol' folder_ and giving it a run.

**Comment by skprabhanjan** [comment]
@roblourens , can i take a look at this ? 
If so then some code pointers would be helpful , Thanks :)

**Comment by lramos15** [comment]
Should fix the message 'You have not yet opened a folder. Only open files are currently searched - Open Folder' makes it sound like my searches will fail with no folder open, but as long as I plug a path in it works like the issue specifies. Maybe add something along the lines of 'unless a path is specified'

**Comment by roblourens** [comment]
I could say, "You have not opened or specified a folder, ..." I'm not sure if that's too wordy, I don't think a warning message has to explain how the whole thing works, what do you think?

**Comment by roblourens** [comment]
Make sure you reopen issues when they are "verification-found"

**Comment by lramos15** [comment]
> I could say, "You have not opened or specified a folder, ..." I'm not sure if that's too wordy, I don't think a warning message has to explain how the whole thing works, what do you think?

This would work. How would it work once I specified a folder but haven't hit send would the message go away due to having a valid path?

**Comment by roblourens** [comment]
No, I think it should just be updated once when the search actually runes

**Comment by lramos15** [comment]
That seems like it would work, it would at least imply that the action being performed would work.

**Comment by skprabhanjan** [comment]
Now it makes more sense !
